snuba/consumers/consumer_builder.py

```diff
@@ -267,6 +270,13 @@ def __build_streaming_strategy_factory(
 
         return strategy_factory
 
+    def close_producers(self) -> None:
```

Comment:
Let's just call this `close()`? I think the fact that it is closing producers is an internal detail of the consumer builder.